src/redisenterprise/azext_redisenterprise/manual/_help.py

```diff
@@ -0,0 +1,278 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['redisenterprise operation-status'] = """
+    type: group
+    short-summary: Manage Redis Enterprise operation status
+"""
+
+helps['redisenterprise operation-status show'] = """
+    type: command
+    short-summary: Get the status of an operation
+    examples:
+      - name: Get the status of an operation.
+        text: |-
+               az redisenterprise operation-status show --operation-id "testoperationid" --location "West US"
```

Comment:
do we need to expose such commands to customers?

Reply:
We expose this in PowerShell, so I thought I would expose them in Azure CLI as well.  I'm not sure if they're required or not, but our REST API shows that they're available to call.